legislators-social-media.yaml

```diff
@@ -1879,6 +1879,11 @@
     youtube_id: UCqMici9I7nz_i-PN7eSYjwQ
     twitter_id: 24745957
     instagram: rephankjohnson
+- id:
+    bioguide: J000300
+    govtrack: 412741
+  social:
+    twitter: sendougjones
```

Comment:
Doug Jones' official site isn't set up yet, but the Verified mark, and the bio/link on the Twitter profile that goes to the official site, makes me comfortable this isn't a campaign site. :+1: